deel-2-flexbox-layout/opdracht-4/styles/styles.css

```diff
@@ -1 +1,80 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+
+body {
+    display: flex;
+    flex-direction: row;
+    width: 100vw;
+    height: 100vh;
+}
+
+.outer-col-1{
+    flex-direction: column;
+    flex-basis: 40vh;
+    flex-shrink: 0;
+    flex-grow: 0;
+
+
+}
+
+.outer-col-2{
+    display: flex;
+    flex-direction: column;
+    flex-basis: 80vw;
+    flex-shrink: 1;
+    flex-grow: 1;
+
+
+
+
+}
+
+header{
+    display: flex;
+    flex-direction: row;
+    flex-basis: 100px;
+    flex-shrink: 0;
+    flex-grow: 0;
+
+
+}
+
+.inner-row{
+    display: flex;
+    flex-direction: row;
+    flex-basis: 50%;
+    flex-wrap: nowrap;
+    flex-shrink: 1;
+    flex-grow: 1;
+
+}
+
+.inner-col{
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    flex-shrink: 1;
+    flex-grow: 1;
+
+
+}
+
+aside{
+    flex-basis: 350px;
+    flex-shrink: 0;
+    flex-grow: 0;
+
+}
+
+footer{
+    flex-basis: 200px;
+    flex-shrink: 0;
+    flex-grow: 0;
+
+}
+
+article{
+    flex-basis: 650px;
```

Comment:
Nova gebruikt hier auto. Ik heb het aangepast in jouw code en auto geeft hetzelfde resultaat als wat jij had. Ik denk dat auto in dat geval de voorkeur heeft boven een specifieke maat in px.